### ISSUE TITLE
"Remember last playback position" remembers everything

### ISSUE THREAD

**Issue by 2V3EvG4LMJFdRe** [iina/iina#4357]
**System and IINA version:**

- macOS High Sierra
- IINA 1.3.1 Build 133

**Expected behavior:**

"Remember last playback position" remembers last playback position.

**Actual behavior:**

"Remember last playback position" remembers everything - equalizer, mute status, video cropping, you name it.
 
**How often does this happen?**

Always and for many years now. I've opened multiple issues over the year that all trace back to this problem (https://github.com/iina/iina/issues/2579 https://github.com/mpv-player/mpv/issues/6907 etc)

**Comment by 2V3EvG4LMJFdRe** [comment]
I finally found the solution to this problem, at least for my use case. "Remember last playback position" in fact remembers a lot more options than just playback position, this is the behaviour by design and it's described as such on the mpv documentation. If the user wants to only "Remember last playback position" you actually have to add `watch-later-options-clr` to your mpv.conf, which removes the saving of other settings from the watch_later files.

I'm leaving the issue open to bring attention to this issue. I propose the box is renamed to something else like "Remember playback position and settings".

**Comment by low-batt** [comment]
Yes, the name of the IINA setting needs to change. That it only mentions position is definitely misleading.

There was a thread race condition that could cause the watch later file to sometimes not be saved when quitting IINA. That has been fixed.

There are other issues in this area. Certain saved settings are not being properly restored such as the selected subtitle (`sid`).

What attributes mpv will save is controlled by the [watch-later-options](https://mpv.io/manual/stable/#options-watch-later-options) property. IINA uses the mpv default value which saves:
- osd-level
- speed
- edition
- pause
- volume
- mute
- audio-delay
- fullscreen
- ontop
- border
- gamma
- brightness
- contrast
- saturation
- hue
- deinterlace
- vf
- af
- panscan
- aid
- vid
- sid
- sub-delay
- sub-speed
- sub-pos
- sub-visibility
- sub-scale
- sub-use-margins
- sub-ass-force-margins
- sub-ass-vsfilter-aspect-compat
- sub-ass-override
- ab-loop-a
- ab-loop-b
- video-aspect-override

If you want to check what the mpv [Watch Later](https://mpv.io/manual/stable/#watch-later) feature has saved, this [Terminal](https://support.apple.com/guide/terminal/welcome/mac): session shows me examining the contents of the watch later file for the movie I was playing: . 
```bash
low-batt@gag ~$ md5 -qs '/Users/low-batt/Movies/resume-failure.mp4' | xargs -I % cat ~/Library/Application\ Support/com.colliderli.iina/watch_later/%
start=215.066667
pause=yes
brightness=-41
sub-scale=2.000000
low-batt@gag ~$ 
```

Use this command to check the file you are playing, substituting the _full pathname_ to your file as I did in the above example:
```bash
md5 -qs 'put full path to file here' | xargs -I % cat ~/Library/Application\ Support/com.colliderli.iina/watch_later/%
```

**NOTE** that the full pathname to the file **MUST** exactly match or the MD5 hash will not be correctly calculated.
